app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :shelter
+
+  validates_presence_of :title
+  validates_presence_of :rating
+  validates_presence_of :content
+  validates_presence_of :image, :allow_blank => true
```

Comment:
Can you explain how the allow_blank works?

Reply:
It allows for no image to be added.